Automattic/wc-calypso-bridge
# decrease padding on table per page select in Calypso

Closes #492 

This PR adjusts the padding and location of the chevron in the table per page drop down.

Note that in master the `calypsoify-active` body class is being inserted whether calypsoify is active or not (https://github.com/Automattic/wc-calypso-bridge/pull/494#issuecomment-516511607).

### Testing
- Go to an analytics table that has more than 100 entries
- Verify that the full `100` string is visible in the per page dropdown.
